src/apps/integrations/crud/integrations.py

```diff
@@ -20,3 +22,11 @@ async def create(self, schema: IntegrationsSchema) -> IntegrationsSchema:
                 type=schema.type, applet_id=schema.applet_id
             )
         return new_integrations
+
+    async def retrieve(self, schema: IntegrationsSchema) -> IntegrationsSchema:
```

Comment:
If the integration doesn't exist this results in an Unhandled error
![CleanShot 2024-08-20 at 13 52 43](https://github.com/user-attachments/assets/eb1ac141-6107-4413-9c78-d2918e296937)